Gemfile

```diff
@@ -165,7 +165,7 @@ group :development do
   # Static analysis
   gem "brakeman"
 
-  gem "listen", ">= 3.0.5", "< 3.3"
+  gem "listen", ">= 3.0.5", "< 3.4"
```

Comment:
Should we bother specifying versions in the `Gemfile`?

`Gemfile.lock` will lock the version anyway until we `bundle update` (update all gems), `bundle update gem-name` (update both gem and dependencies), or `bundle update --source gem-name` (update exclusively the gem).

Reply:
I've always said that. That's what the Gemfile.lock is for. We should only pin versions that we need to stick on a specific version for some reason. I think it's one of those polarising single vs double quote type of arguments. If we agree to go with that we should let this one through and do it in another PR though.